app/web/features/communities/events/CommunityEventsList.tsx

```diff
@@ -69,10 +62,7 @@ export default function CommunityEventsList({
         ) : hasAtLeastOnePage(data, "eventsList") ? (
           data.pages
             .flatMap((page) => page.eventsList)
-            .filter((event) => !event.isCancelled)
-            .map((event) => (
-              <CommunityLongEventCard event={event} key={event.eventId} />
-            ))
+            .map((event) => <LongEventCard event={event} key={event.eventId} />)
```

Comment:
I think you removed the `.filter` by mistake here when you were switching out the `.map` return!